src/main/java/toughasnails/temperature/MaterialTemperatureData.java

```diff
@@ -0,0 +1,7 @@
+package toughasnails.temperature;
+
+public class MaterialTemperatureData {
+
+	public float FIRE = 1.0F;
+	
```

Comment:
This seems incomplete to me... Shouldn't you have a field for temperature and for the material itself? Also in terms of naming, we generally don't use caps unless they are either static final or it is an enum. This is *usually* consistent across most styles I've seen.